changelogs/fragments/XXXX-add-from-csv-filter.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
I think this copy should go ;-)

Reply:
Yes, sorry change was staged, but not committed.